config/docker-compose.yml.tim

```diff
@@ -29,6 +27,4 @@ services:
       - "/bin/sh"
       - "-c"
       - |
-          /opt/sd/launch --api-uri {{api_uri}} --store-uri {{store_uri}} --emitter /opt/sd/emitter {{build_id}} &
-          /opt/sd/logservice --emitter /opt/sd/emitter --api-uri {{api_uri}} --store-uri {{store_uri}} --build {{build_id}} &
-          wait $$(jobs -p)
+          /opt/sd/run.sh "{{token}}" {{api_uri}} {{store_uri}} $SD_BUILD_TIMEOUT {{build_id}}
```

Comment:
can you double quote these shell variables as well ?

Reply:
Fixed it.